src/Bridge/Symfony/Bundle/Resources/config/graphql.xml

```diff
@@ -65,9 +65,14 @@
             <argument type="service" id="api_platform.graphql.resolver.resource_field" />
             <argument type="service" id="api_platform.graphql.types_factory" />
             <argument type="service" id="api_platform.filter_locator" />
+            <argument type="service" id="api_platform.graphql.query_resolver_locator" />
             <argument>%api_platform.collection.pagination.enabled%</argument>
         </service>
 
+        <service id="api_platform.graphql.query_resolver_locator" class="Symfony\Component\DependencyInjection\ServiceLocator">
```

Comment:
Add it with the resolvers?

Reply:
Then it would be before the `$typesFactory` argument, which is not nullable right now. But the `$queryResolverLocator` is. Should I just make it required? (what was the consesus on the null / required discussion?)

Reply:
That's not what I meant :slightly_smiling_face:
I was just saying the service definition should be with the other resolver service definitions (below `<!-- Resolvers -->`).
But to answer your question, since we decided that the `typesFactory` is required, there is already a breaking change. Therefore I think you can make it required and move it before the `typesFactory`.

Reply:
Ah I see it now, makes sense. I'll make both changes 👍